docker/Dockerfile

```diff
@@ -1,13 +1,16 @@
-# Using the same image as we use in CircleCI to avoid transfer costs
-FROM cimg/openjdk:14.0.2 AS install
+FROM ubuntu:22.04 AS install
 
 # Use latest stable release here
-ENV CONFLUENT_VERSION=7.3.1
+ENV CONFLUENT_VERSION=7.5.2
 
 USER root
 WORKDIR /install
 COPY install.sh /tmp/
+RUN apt-get update && apt-get install curl -y
 RUN /tmp/install.sh && rm /tmp/install.sh
+RUN rm /install/share/java/schema-registry/zookeeper-3.6.3.jar /install/share/java/schema-registry/zookeeper-jute-3.6.3.jar && \
```

Comment:
critical vulnerability `CVE-2023-44981` in zookeeper version 3.6.3. upgrading it to 3.8.3.